### ISSUE TITLE
README missunderstood

### ISSUE THREAD

**Issue by ytiruces** [iotaledger/iota-core#960]
Hi!

In [README.md](https://github.com/iotaledger/iota-core?tab=readme-ov-file#iota-core---the-iota-20-node) I see:

> In this repository you will find the following branches:
> 
> - production: this branch contains the latest released code targeted for the [IOTA mainnet](https://iota.org/)
> - staging: this branch contains the latest released code targeted for the [Shimmer network](https://shimmer.network/)
> - develop: default branch where all development will get merged to. This represents the next iteration of the node.

But there are no [branches](https://github.com/iotaledger/iota-core/branches) `production` and `staging`.

**Comment by muXxer** [comment]
yeah because this version of the IOTA protocol is not live on any of the two mentioned networks.

**Comment by ytiruces** [comment]
Please direct me where the code of the mentioned networks is.

**Comment by Dr-Electron** [comment]
The current node software for mainnet is [Hornet](https://github.com/iotaledger/hornet). [Docs](https://wiki.iota.org/hornet/welcome)

**Comment by ytiruces** [comment]
Thank you.  Still, it seems the README of this repo needs fixing.
